oas_docs/output/kibana.serverless.yaml

```
@@ -79648,6 +79656,11 @@ components:
           enum:
             - delete
           type: string
+        gap_fill_statuses:
+          description: Aggregated gap fill statuses to filter rules with gaps by status (used together with gaps_range_*).
```

I'm not sure what to suggest for this yet because I need more background on the "aggregated gap fill statuses" phrase. I'm also not clear on what the text in the parenthesis is actually communicating. Are the returned results somehow affected by when gaps start and end for the specified rules?

---

@nkhristinin @nastasha-solomon how about something like:

```suggestion
          description: The highest priority status of the gap fills in question. Priority is `unfilled > in_progress > filled`. Used to filter rules to those with a particular gap fill status.
```